docs/documentation/publishing-on-heroku-terminal.md

```diff
@@ -61,7 +61,7 @@ Make sure any changes you've made to your prototype have been committed to git.
 
 From your prototype folder:
 ```
-git push heroku master
+git push heroku main
```

Comment:
I wonder if we should make it clear that this is an example, when you want to push the contents of your main branch to heroku? Just thinking if people aren't so familiar with `main` as they are with `master`, they might not immediately understand what the command is doing and be able to replace "main" for their own use case, e.g: if their prototype still uses a master branch.

Reply:
Yeah that's a tricky one... They also might not know what a branch is though. @EoinShaughnessy do you have any thoughts?

Reply:
I guess it depends on what roles use these docs. If it's only developers who use them, then we might not have to explain branches.

The next sentences after the command are:

> This will push your work to Heroku. Deploying may take a minute or so.

Maybe we could add something like this after them:

> In this example, `main is the name of the default branch for the repo. So you may need to run the command with your own default branch name instead of `main`.

Reply:
> I guess it depends on what roles use these docs. If it's only developers who use them, then we might not have to explain branches.

I think that these docs are aimed at prototype kit users, who might not be developers. I've heard that designers use it a lot, possibly also content designers and product managers?

Reply:
Ah, interesting re variety of roles. The team devs will have a better idea than me of what knowledge it's fair to assume that Kit users have.

Reply:
I think the "In this example..." text you suggested @EoinShaughnessy meets the suggestion that @vanitabarrett made, so I'll incorporate it.

Reply:
sorry, late to the conversation. I think possibly the best thing to do here is leave the instruction as `master` as that is the default branch they will get on their git project, unfortunately

Reply:
> sorry, late to the conversation. I think possibly the best thing to do here is leave the instruction as `master` as that is the default branch they will get on their git project, unfortunately

Ah, of course, because the guide recommends setting up repositories using the git command line.

We could add a line explaining to users how to change the name of the default branch; we added a [line to that effect to the prototype kit training](https://github.com/alphagov/prototype-kit-training/pull/58/commits/3cb94e573bded2472829c23a5d6a3dd87c8511c3).

Reply:
ah, perfect yes that line looks good

Reply:
Commit https://github.com/alphagov/govuk-prototype-kit/pull/1018/commits/0ea55f998a3b5b6b86c0a4c323a4eba3a2dccdb8 adds a step to the "Setting up Git in the terminal" page to rename the default branch. GitHub Desktop should use `main` by default already so I didn't update the page "Storing your code online with GitHub and GitHub Desktop".